tests/results/SourceRangeTest.py

```diff
@@ -86,6 +86,18 @@ def test_json(self):
                                       end_line=4).__json__(use_relpath=True)
         self.assertEqual(uut['start'], self.result_fileB_line2)
 
+    def test_str(self):
+        uut = str(SourceRange.from_values('test_file', 1, 1, 2, 1))
+        compare = abspath('test_file') + ': L1 C1 : L2 C1'
+        self.assertEqual(uut, compare)
+
+        uut = str(SourceRange.from_values('test_file', 1, None, 2, None))
+        compare = abspath('test_file') + ': L1 : L2'
+        self.assertEqual(uut, compare)
+
+        uut = str(SourceRange.from_values('test_file', None, None, None, None))
+        self.assertEqual(uut, abspath('test_file'))
```

Comment:
> ... or maybe just use column number 1 then? --> L3C1 : L4C5

nvm this would be inconsistent when omitting the end-column, as then we would have to read the file to check out which one is the last column^^ We don't want that^^

So then `L3 : L4C5` is better or `L3C5 : L4`